echo/index.js

```diff
@@ -181,13 +181,32 @@ function Client() {
     debug('request options', requestOptions);
 
     request.post(requestOptions, function onResp(err, response, body) {
-      var statusCode = response && response.statusCode
-        ? response.statusCode
-        : 0;
-
-      if (err || parseInt(statusCode / 100) !== 2) {
-        error('[echoClient] addEvents error', { err: err, body: body, statusCode: statusCode });
-        callback(err || 'error response status code: ' + statusCode);
+      if (err) {
+        var errorResponse = {
+          statusCode: 0,
```

Comment:
the original code would set the status code to the response.statusCode if it existed, else it would default it to zero. Is there a reason why we wouldn't want to do that now? It looks like we are just defaulting to zero.